controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/application/JobControllerApiHandlerHelperTest.java

```diff
@@ -161,7 +161,7 @@ private void compare(HttpResponse response, String expected) {
 
     private void assertFile(HttpResponse response, String resourceName) {
         try {
-            Path path = Paths.get(getClass().getClassLoader().getResource(resourceName).getPath());
```

Comment:
Just curious, why this change?

Reply:
IntelliJ sometimes gets confused which class path to use, which causes tests that use this method to fail with NPE. I think it happens after I install the module locally with `mvn install`, which I sometimes do to check for breaking API between this and internal repo.